docs/performance-guide.md

```diff
@@ -25,150 +54,205 @@ guide intends to answer the following questions:
+<a name="evaluation"></a>
+### Find a proper SKU
+
+How to evaluate the performance inbound/outbound capacity or how to find which tier is suitable for a specific use case?
+
+Let us simplify the evaluation first by assuming some default settings: Websockets is applying, message size is 2048 bytes, sending message every 1 second, and apply default mode.
+
+We assume the app server is powerful enough and is not the performance bottleneck. Then we can check the maximum inbound and outbound bandwidth for every tier.
+
+<a name="quick_evaluation"></a>
+#### Quick evaluation
+
+For the inbound message limit, let us choose **echo** which has the lowest routing cost. For the outbound message limit, broadcast is the good use case. The following table lists the maximum inbound and outbound message bandwidth (bytes per second).
+
+|                                   | Unit1 | Unit2 | Unit5 | Unit10 | Unit20 | Unit50 | Unit100 |
+|-----------------------------------|-------|-------|-------|--------|--------|--------|---------|
+| Connections                       | 1,000 | 2,000 | 5,000 | 10,000 | 20,000 | 50,000 | 100,000 |
+| Inbound bandwidth (byte/s) | 2M    | 4M    | 10M   | 20M    | 40M    | 100M   | 200M    |
+| Outbound bandwidth (byte/s) | 8M    | 8M    | 20M    | 40M    | 80M    | 200M    | 400M    |
+
+The inbound bandwidth and outbound bandwidth formulas:
+```
+  Inbound bandwidth = inboundConnections * messageSize / sendInterval
+  Outbound bandwidth = outboundConnections * messageSize / sendInterval
+```
+
+*inboundConnections*: the number of connections sending message
+
+*outboundConnections*: the number of connections receiving message
+
+*messageSize*: the size of a single message (average value). For small message whose size is less than 1024 bytes, it has the similar performance impact as 1024 byte message.
+
+*sendInterval*: the time of sending one message, typically it is 1 second per message, which means sending 1 message every second. Smaller sendInterval means sending more message in given time period. For example, 0.5 second per message means sending 2 message every second.
+
+*Connections* is the ASRS committed maximum threshold for every tier. If the connection number is increased further, it will suffer from connection throttling.
+
+*Inbound bandwidth* and *Outbound bandwidth* are the message size per second. Here 'M' means megabyte for simplicity.
+
+<a name="mixed_evaluation"></a>
+#### Evaluation for mixed use cases
+
+The real use case is more complex. It typically mixes the 4 basic use cases together: **echo**, **broadcast**, **send to group**, or **send to connection**. The methodology used to evaluate the capacity is divide the mixed use cases to 4 possible basic use cases, calculate the capacity separately, and sum them.
+
+For every basic use case, **calculate the maximum inbound and outbound message bandwidth** using the above formulas.
```

Comment:
Then what? The inbound and outbound should not exceed the limit the above table?

Reply:
Yes, the table wants to show the maximum inbound/outbound.